{"error":"SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint \"project_social_account_project_id_location_identity_unique\"\nDETAIL:  Key (project_id, location_identity)=(3, 109239870969734) already exists. (SQL: insert into \"project_social_account\" (\"project_id\", \"social_account_id\", \"location_identity\", \"updated_at\", \"created_at\") values (3, 29, 109239870969734, 2020-10-24 18:07:43, 2020-10-24 18:07:43) returning \"id\")"}